yvalenta/spec/connection_spec.rb

```diff
@@ -0,0 +1,78 @@
+require "spec_helper"
+
+describe "Connection" do
+  let(:connection) { Connection.new }
+  let(:regex) { /[0-9a-f]{ 8 }-[0-9a-f]{ 4 }-[0-9a-f]{ 4 }-[0-9a-f]{ 4 }-[0-9a-f]{ 12 }/}
+
+  describe "#add" do
+    it 'must return an uuid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.